OmniBLE/PumpManagerUI/Views/OmniBLESettingsView.swift

```diff
@@ -403,15 +420,27 @@ struct OmniBLESettingsView: View  {
                 }
                 NavigationLink(destination: BeepPreferenceSelectionView(initialValue: viewModel.beepPreference, onSave: viewModel.setConfirmationBeeps)) {
                     HStack {
-                        FrameworkLocalText("Confidence Reminders", comment: "Text for confidence reminders navigation link").foregroundColor(Color.primary)
+                        FrameworkLocalText("Confidence Reminders", comment: "Text for confidence reminders navigation link")
+                            .foregroundColor(Color.primary)
                         Spacer()
                         Text(viewModel.beepPreference.title)
                             .foregroundColor(.secondary)
                     }
                 }
+                if  (allowDebugFeatures) {
```

Comment:
Extra space after the "if" keyword.

I'd much prefer that Silence Pod support not be hidden if allowDebugFeatures isn't true as it's a very useful basic feature that should always be available. Also, if it is sometimes hidden and other times not, then if the OmniBLEPumpManagerState.silencePod state variable gets set to true while allowDebugFeatures is true, then this state variable can never be reset by the user and all future pods will never give audible pod alerts if subsequent versions of Loop don't also have allowDebugFeatures set to true.

Reply:
I was thinking of the requirement needed with LoopKit / MockKit which is also used by Tidepool. But the DIY pump managers will not be shared with Tidepool. So I reverted the commit that added the flag.